### DISCUSSION TITLE
Telecamera wifi , no onvif come prendere video ?

### DISCUSSION THREAD

**Discussion by ziopeppus2** [indomus/forum#922] [Hardware & Firmware]
Salve a tutti , il titolo  è un po' lungo e riassume gran parte della mia domanda.
Spiegandomi meglio : 

Ho 3 telecamere wifi di brand diversi ,  sono tutte indipendenti , nel senso che funzionano registrando il video su una scheda SD da posizionare al loro interno e si può visualizzare il video in streaming o registrato tramite la loro app. Le 3 telecamere sono tutte diverse tra loro e hanno 3 app diverse da dove si controllano.

Nessuna delle 3 riporta la dicitura Onvif o RTSP , mi chiedevo se è possibile comunque estrapolare il video , dato che passano sul mio router. 

Ho iniziato a fare degli esperimenti da quella che mi sembra la più insidiosa  

Con nmap , ho cercato l'ip e le porte aperte della telecamera.  Mi risulta che su questa siano aperte la porta tcp 80 e tcp 10000 , ma sinceramente non so come entrarci. 
Mettendo il link direttamente nel browser nel caso della tcp 80 mi da subito pagina non trovata , mentre nel caso della tcp 10000 parte un caricamento infinito. 

Su internet purtroppo non riesco a trovare nulla su questo argomento specifico. Se qualcuno mi potesse dare qualche dritta anche sulle prove da poter fare ne sarei molto grato.

**Comment by indomus** [comment]
No link, thx

**Comment by poliant** [comment]
ciao,
la vedo difficile se non sai quali sono le porte che funzionano.
Prova ad utilizzare l'app android tinycam e fare una ricerca automatica delle cam e se le trova la app di da info aggiuntive quali porta e protocollo. Una volta fatto ciò prova ad entrare nella cam con un browser ed analizzare i comandi (e.g. premendo F12) così dovresti essere in grado di vedere qual è il percorso per accedere al video e da li poi puoi capire/decidera quale integrazione serve
